### PR TITLE
Issue/8646 quick start completed view

### LINKED ISSUE #8646: Quick Start: Add Collapsed Completed Views
Add views to the full-screen dialog interface when all tasks are completed and the expandable list section is collapsed.  See the mockups below for illustration.

Customize Your Site|Grow Your Audience
-|-
v2_list_customize_collapsed_completed_design|v2_list_growth_collapsed_completed_design

### PR DESCRIPTION
**Pull request by khaykov**
Fixes #8646

This PR adds a "Quick Start Completed" view that appears when all tasks in a list are completed, and the list is collapsed.

Test 1:
- Open Quick Start task list when all the tasks are completed.
- Notice Completed View.
- Toggle the completed tasks list and see that Completed View fades in and out.

Test 2:
- While completed task list is collapsed, skip all the tasks in the list.
- After the last task is skipped notice that the Completed View appears.

Test 3:
- Confirma that AEV illustration are different for GROW and CUSTOMIZE lists.

Visuals (clicking on gif opens HD version):

[Image from Gyazo](https://gyazo.com/1af27c5d7eaa321a4e4199be7f417f20)

[Image from Gyazo](https://gyazo.com/beafecd0526211b768435e89e5f6196f)

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartFullScreenDialogFragment.java [hunk 1]
```diff
@@ -139,9 +153,33 @@ public void onSkipTaskTapped(QuickStartTask task) {
...
+            if (uncompletedTasks.isEmpty() && !mQuickStartAdapter.isCompletedTasksListExpanded()) {
+                toggleCompletedView(true);
+            }
+        }
+    }
+
+    @Override
+    public void onCompletedTasksListToggled(boolean isExpanded) {
+        if (!mQuickStartStore.getUncompletedTasksByType(AppPrefs.getSelectedSite(), mTasksType).isEmpty()) {
+            return;
```
[the remark this answers is not in the document]
Comment (khaykov) [reply WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartFullScreenDialogFragment.java]: You are right! I used to have more code there but removed it at some point :)

File: WordPress/src/main/java/org/wordpress/android/ui/ActionableEmptyView.kt [hunk 2]
```diff
@@ -104,4 +101,13 @@ class ActionableEmptyView : LinearLayout {
         params.topMargin = topMargin
         layout.layoutParams = params
     }
+
+    fun setImage(imageResource: Int) {
```
[the remark this answers is not in the document]
Comment (khaykov) [reply WordPress/src/main/java/org/wordpress/android/ui/ActionableEmptyView.kt]: Got it! Still feels awkward directly accessing variables from java :)

**Comment by khaykov** [comment]
Thanks for taking a look, @theck13 ! Hope I addressed all the issues. Your attention to details is superb as always :)

**Comment by iamthomasbishop** [comment]
Looks good to me, nice work @khaykov! I'll let @SylvesterWilmott have the final say, but nothing stands out to me as off. 👏

**Comment by khaykov** [comment]
@SylvesterWilmott Yes, I think we can. I'll try it out and show you how it looks so we can submit a patch if necessary.

**Pull request merged**